src/index.css

```diff
@@ -0,0 +1,182 @@
+@import url('https://fonts.googleapis.com/css2?family=Righteous&family=Russo+One&display=swap');
```

Comment:
- [ ] In the these sections, the content does not fit the screen:

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/016ec8fe-ff1e-4e37-a69a-84317dc240fe)

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/1af42819-a861-41c2-b9c3-45c7c38bf13b)

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/b5023a54-2429-43de-bd48-296f455adf26)

Kindly touch up your styling so the content fits the screen.

_note: this was tested multiple times on both Chrome and Firefox_